apps/demos/Demos/DropDownButton/Overview/Angular/app/app.component.html

```diff
@@ -14,25 +14,38 @@
   </div>
 
   <div class="dx-field">
-    <div class="dx-field-label"> Custom main button action </div>
+    <div class="dx-field-label">Custom template and actions</div>
     <div class="dx-field-value">
       <dx-drop-down-button
+        id="two-sections"
         [splitButton]="true"
         [useSelectMode]="false"
-        text="Sandra Johnson"
-        icon="../../../../images/gym/coach-woman.png"
+        text="Olyvia Peyton"
```

Comment:
Seems, it's a typo. In our data we use Name: 'Olivia Peyton' 
<img width="184" alt="image" src="https://github.com/DevExpress/DevExtreme/assets/15689080/ab89cec4-7bdd-4726-bd01-2522a744821a">

Reply:
Regarding this, should I follow Olivia (in our data) or in the design (Olyvia)

Figma - https://www.figma.com/file/792Fik8qn6E3laq9CfFJfW/Components?type=design&node-id=5409-51951&mode=design&t=uSCouvloGwLw7K12-0

Reply:
@vladaskorohodova

Reply:
The correct name is Olivia with an "i".

Reply:
Okay, thanks! We'll just fix the tests then refactor it.